configs/README.md

```diff
@@ -0,0 +1,264 @@
+# Maestro Config
+
+The following guide documents the options available in the `maestro.config` yaml file that is provided to maestro on startup.
+
```

Comment:
Maybe document the expected structure first, perhaps providing an example minimal config.

> `---`
> ` config_end: true`